src/languages/en.ts

```diff
@@ -657,8 +656,8 @@ export default {
             `changed the distance to ${newDistanceToDisplay} (previously ${oldDistanceToDisplay}), which updated the amount to ${newAmountToDisplay} (previously ${oldAmountToDisplay})`,
         threadRequestReportName: ({formattedAmount, comment}: ThreadRequestReportNameParams) => `${formattedAmount} ${comment ? `for ${comment}` : 'request'}`,
         threadSentMoneyReportName: ({formattedAmount, comment}: ThreadSentMoneyReportNameParams) => `${formattedAmount} sent${comment ? ` for ${comment}` : ''}`,
-        tagSelection: ({tagName}: TagSelectionParams) => `Select a ${tagName} to add additional organization to your money.`,
-        categorySelection: 'Select a category to add additional organization to your money.',
+        tagSelection: 'Select a tag to better organize your spend.',
```

Comment:
Why are we no longer using the tagName? If the field is called "Department" for example, they won't know what "tag" means right?

Reply:
See the linked conversation, but 2 reasons:

- The tag list can be called `Departments`, `Select a Departments to better organize your spend.` is not english
- `Departments` is present at the top of the view